packages/falter-berlin-dynamic-banner/files/refresh_banner.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Get dynamic information and construct a new banner. This script
+
+HOSTNAME=$(uci get system.@system[0].hostname)".olsr"
+IPADDR=$(uci get network.wireless0.ipaddr)
+UPTIME=$(uptime | cut  -d' ' -f4-5 | cut -d',' -f1)
+FREEFL=$(df -h | grep " /overlay" | sed -E -e s/[[:space:]]+/\;/g | cut -d';' -f4 )
+SYS_LOAD=$(uptime | rev.sh | cut -d':' -f1 | rev.sh | sed -E -e s/[[:space:]]+/\/ )
```

Comment:
```uptime | cut -d ',' -f 0 | cut -d ' ' -f 4-``` should work just as well, and the extra script "rev.sh" would not be needed.

Reply:
Examples:
```
root@mynode:~# uptime | cut -d ',' -f 0 | cut -d ' ' -f 4-
29 days
```
or after a fresh reboot
```
root@mynode:~# uptime | cut -d ',' -f 0 | cut -d ' ' -f 4-
1 min
```

Reply:
include into 254237d162c550c54da2e9b187ef92d114b9201d